wrappers/golang/curves/bls12377/tests/msm_test.go

```diff
@@ -184,16 +184,18 @@ func TestPrecomputeBase(t *testing.T) {
 			_, e := precomputeOut.Malloc(points[0].Size()*points.Len()*int(precomputeFactor), points[0].Size())
 			assert.Equal(t, e, cr.CudaSuccess, "Allocating bytes on device for PrecomputeBases results failed")
 
-			e = msm.PrecomputeBases(points, precomputeFactor, 0, &cfg.Ctx, precomputeOut)
+			cfg.PrecomputeFactor = precomputeFactor
+			cfg.PointsSize = int32(points.Len())
+			cfg.ArePointsOnDevice = points.IsOnDevice()
```

Comment:
Similar to the previous comment, this should be checked and updated in `core.PrecomputeBasesCheck` which is called in `msm.PrecomputeBases`

Reply:
ok, will fix, what about the rust test? is it important there too?

Reply:
Yes, there as well. `PrecomputeFactor` is fine to update manually as it is public but `points_size` is not public so it shouldn't be updated manually